pom.xml

```diff
@@ -49,6 +49,7 @@
         <maven.version>3.6.0</maven.version>
         <site.jira.version.id>12344664,12347157,12348422,12350010</site.jira.version.id>
```

Comment:
Can you also please add the new release here?

Reply:
will do outside this PR (because not really related to the parent update)